### PR TITLE
Add exception hunter

### PR DESCRIPTION
**Pull request by brunvez**
#### Description:

Adds [ExceptionHunter](https://github.com/rootstrap/exception_hunter) to track errors into the database

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  ExceptionHunter.routes(self)
```
Comment (santib) [inline config/routes.rb:2]: minor comment: maybe we can move engine routes at the bottom of the file?

**Comment by CR1AT0RS** [comment]
Adding exception hunter breaks the "apartment" gem compatiblity.

**Comment by CR1AT0RS** [comment]
@brunvez @santib 
What's the URL to access the exception hunter? 
I've tried:
http://localhost:3000/exception_hunter/errors
http://localhost:3000/admin/exception_hunter/errors

No routes found!

**Comment by brunvez** [comment]
> @brunvez @santib
> What's the URL to access the exception hunter?
> I've tried:
> http://localhost:3000/exception_hunter/errors
> http://localhost:3000/admin/exception_hunter/errors
> 
> No routes found!

Hey @CR1AT0RS it's `http://localhost:3000/exception_hunter/errors` but you have to enable ExceptionHunter on development. That's done through the initializer at `config/initializers/exception_hunter.rb`. You can set `config.enabled = true || !(Rails.env.development? || Rails.env.test?)` to test it. Any feedback is welcomed!

**Pull request merged**